webtool/gradle/common/src/commonMain/kotlin/Pilot.kt

```diff
@@ -7,4 +7,78 @@ import kotlin.js.JsExport
 @JsExport
 data class Pilot(
     val name: String,
+    val numHouseholds: int,
+    val households: List<Households>,
+    val numCompanies: int,
+    val companies: List<Company>,
+    val numSolarfarms: int,
+    val solarfarms: List<Solarfarm>,
+    val numWindfarms: int,
+    val windfarms: List<Windfarm>,
+    val numBatteries: int,
+    val batteries: List<Battery>,
+    val numHeatStorages: int,
+    val heatStorages: List<HeatStorage>,
 )
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+data class Households(
+    val type: String,
+    val numHouseholds: int,
+    val hasPVRatio: double,
```

Comment:
`int` -> `Int`
`double` -> `Double`

Dan lukt het te compileren

Reply:
in de anylogic modellen zouden we waarschijnlijk iets als hasPVRatio_r of numHouseholds_n doen, al zit die _r en _n hier ook in de naam.